src/haz3lweb/init/Init.re

```diff
@@ -1,7 +1,12 @@
 open Haz3lcore;
 
 let startup: PersistentData.t = {
-  scratch: (0, List.init(8, _ => Zipper.init() |> PersistentZipper.persist)),
+  scratch: (
+    0,
+    List.init(2, i =>
```

Comment:
How many scratch buffers should we start off with? I'd probably just do 1 but I'm open to opinions

Reply:
yeah just one named Empty Scratchpad 1 would be okay, and maybe number from there?